mitodl/ocw-hugo-themes
# fix instructor json

#### Pre-Flight checklist


#### What are the relevant tickets?
https://github.com/mitodl/ocw-hugo-themes/issues/244

#### What's this PR do?
In https://github.com/mitodl/ocw-studio/pull/646, we updated `ocw-studio` to serialize the `type` property on `WebsiteContent` objects as `content_type` in Hugo front matter to avoid a conflict with the Hugo`type` property, which forces Hugo to use a specific set of layouts for a given piece of content.  In https://github.com/mitodl/ocw-hugo-themes/pull/186, we set up templates in `ocw-hugo-themes` to render a static JSON API for instructor content and set up the templates to be in `course/layouts/instructor` because instructors were being assigned `type: instructor`.  Since this property is now in `content_type` and the instructor content is stored in `content/instructors`, the corresponding JSON layouts need to be in `course/layouts/instructors` to be picked up by default during the Hugo build of `ocw-www`.  This PR simply moves `course/layouts/instructor/single.json.json` to `course/layouts/instructors/single.json.json` to account for this.

#### How should this be manually tested?
 - Clone `ocw-to-hugo` and make sure you are set up to download courses from S3
 - In your `ocw-to-hugo` folder, run `node . -i private/input -o private/output -c course_json_examples/example_courses.json --download --rm`
 - Clone https://github.com/mitocwcontent/ocw-www and https://github.com/mitodl/ocw-hugo-projects
 - Back here in `ocw-hugo-themes`, make sure you have the following in your `.env` file:
   - `STATIC_API_BASE_URL=http://localhost:3000`
   - `WWW_CONTENT_PATH=/path/to/mitocwcontent/ocw-www/`
   - `COURSE_CONTENT_PATH=/path/to/ocw-to-hugo/private/output/`
   - `COURSE_HUGO_CONFIG_PATH=/path/to/ocw-hugo-projects/ocw-course/config.yaml`
 - Run `npm run start:www`
 - In a separate terminal window, also within the `ocw-hugo-themes` folder, run `./package_scripts/build_all_courses.sh`
 - Visit http://localhost:3000/courses/18-06-linear-algebra-spring-2010/ and verify that you see "Instructor: Gilbert Strang" in the course info section
